modules/draco/src/lib/draco-parser.ts

```diff
@@ -351,6 +355,9 @@ export default class DracoParser {
     attribute: DracoAttribute
   ): {value: TypedArray; size: number} {
     const TypedArrayCtor = DRACO_DATA_TYPE_TO_TYPED_ARRAY_MAP[attribute.data_type];
+    if (!TypedArrayCtor) {
+      throw new Error(`DRACO: Unsupported attribute data type ${attribute.data_type}.`);
```

Comment:
Seems that any unknown type would belong to some extra property that does not block rendering. Can we warn and ignore instead of throwing?